Fixed #34112 -- Added async interface to model methods (#16235)
django/django: docs/releases/4.2.txt

What's new in Django 4.2
========================

Asynchronous Model interface
----------------------------

``Model`` now provides an asynchronous interface for some model methods that
user database. These are named as -per the existing synchronous operations but
with an ``a`` prefix, for example ``asave()``, ``adelete()``, and so on.

> see :ref:`async-model-methods` for details.

Reviewer: This is a much smaller change than the async QuerySet API, so I think it should be a list item under the Models heading (line 215). Separate sections are reserved for headline features.

* :class:`~django.db.models.functions.Now` now supports microsecond precision
  on MySQL and millisecond precision on SQLite.

> * ``Model`` now provides an asynchronous interface for some model methods that
>   user database. These are named as -per the existing synchronous operations
>   but with an ``a`` prefix, for example ``asave()``, ``adelete()``, and so on.

Reviewer: The `QuerySet` release note in 4.1 didn’t list every method because it would have been too long. But we’re only adding three methods here, so I think we should list them. Also we should use `:meth:` for references so they work as links, which is a lot more useful for readers.

I suggest:

```suggestion
* ``Model`` now provides asynchronous versions of some model methods that use
  the database, using an ``a`` prefix: :meth:`~Model.adelete`,
  :meth:`~Model.arefresh_from_db`, and :meth:`~Model.asave`.
```

Author: Thanks for the review @adamchainz , but the ref is missing for these methods.
